scripts/Tools/pelayout

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+
+"""
+This utility allows the CIME user to view and modify a case's PE layout.
+
+Interpreted format sequences are:"
+%%  a literal %
+%C  the component name
+%T  the task count for the component
+%H  the thread count for the component
+%R  the PE root for the component
+
+Standard format extensions, such as a field length and padding are supported.
+Example:
+  For NTASKS_ATM=480, NTHREADS_ATM=2 and ROOTPE_ATM, \"%C: %06T/%+H\" will show
+ATM:    480/+2
+Note that since %R was omitted, the root PE was not shown.
+=======================
+If this tool is missing any feature that you need, please go to github and
+create an issue (https://github.com/ESMCI/cime/issues). Detailed
+information is available at https://github.com/ESMCI/cime.
+
+    COPYRIGHT AND LICENSE
+
+        This library is free software; you can redistribute it and/or modify
+        it under the same terms as Python itself.
+"""
+
+from standard_script_setup import *
+
+from CIME.case import Case
+from CIME.utils import expect, convert_to_string
+import textwrap
+import sys
+import re
+
+logger = logging.getLogger("xmlquery")
+
+###############################################################################
+def parse_command_line(args):
+###############################################################################
+    parser = argparse.ArgumentParser(description="Display and/or change the case's PE layout." ,
+                                     formatter_class=argparse.RawDescriptionHelpFormatter,
+                                     epilog=textwrap.dedent(__doc__) )
+    CIME.utils.setup_standard_logging_options(parser)
+
+    # Set command line options
+    parser.add_argument("-format" , "--format",
+                        help="Format the PE layout items for each component (see below)")
+
+    parser.add_argument("-caseroot" , "--caseroot", default=os.getcwd(),
+                        help="Case directory to build")
```

Comment:
This help text doesn't make sense to me for this tool. I'm guessing you copied it from xmlquery, which has the same non-sensical help-text.